react/ViewStack/ModalStack.jsx

```diff
@@ -29,11 +32,16 @@ const ModalStack = ({ children }) => {
         const hasProps = child.length > 1
         const props = hasProps ? child[1] : null
         child = hasProps ? child[0] : child
-        return (
-          <Modal mobileFullscreen key={i} dismissAction={stackPop} {...props}>
-            {child}
-          </Modal>
-        )
+        const shouldWrap = props ? !props.noModalStackWrapper : true
+        if (shouldWrap) {
+          return (
+            <Modal mobileFullscreen key={i} dismissAction={stackPop} {...props}>
+              {child}
+            </Modal>
+          )
+        } else {
+          return React.cloneElement(child, { key: i })
+        }
```

Comment:
Can't we do a bit of magic and detect that the first child is a Modal / Dialog ? And add a forceModalWrapper if this magic/ default behavior is not the one we want ?